python/example/nanogpt_4D_finetune/finetune_4D.py

```diff
@@ -30,11 +30,11 @@
 
```

Comment:
`python` folder should be gone by a picked MR?

Reply:
yes. it is done in the latest commit after incorporating randomness.